src/strings.js

```diff
@@ -1,26 +1,26 @@
-function sayHello (string) {
-  // your code here
-};
+const sayHello = (string) => {
+  return `Hello, ${string}!`;
+}
 
-function uppercase (string) {
-  // your code here
-};
+const uppercase = (string) => {
+  return string.toUpperCase();
+}
 
-function lowercase (string) {
-  // your code here
-};
+const lowercase = (string) => {
+  return string.toLowerCase();
+}
 
-function countCharacters (string) {
-  // your code here
-};
+const countCharacters = (string) => {
+  return string.length;
+}
 
-function firstCharacter (string) {
-  // your code here
-};
+const firstCharacter = (string) =>{
```

Comment:
Should leave a space between the `>` and the `{` opening bracket of your function:
```
const firstCharacter = string => {
```